deploy/crds/kieapp_crd.yaml

```diff
@@ -74,11 +74,11 @@ spec:
                             description: Source for the environment variable's value
                     resources:
                       type: object
-                      anyOf:
-                      - required:
-                        - limits
-                      - required:
-                        - requests
+                      # anyOf:
+                      # - required:
+                      #   - limits
+                      # - required:
+                      #   - requests
```

Comment:
Did you have a problem with these? I had tested these and despite the fact that the top ones caused problems, these seemed to work just fine with oc 3.11

Reply:
With the lab outage, it's difficult to properly validate these. However, I've tested locally and at least with kubectl 1.13 and oc 3.11, the local side validation passes for the trial env with your items uncommented. I think we have the oc 3.10 client in the lab, which I should also try. Please also let me know what you are using, because the version of your client-side oc library is probably what matters most.